### PR TITLE
Improve types for express server

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

File: samples/http-client-examples/server.ts [hunk 1]
```diff
@@ -1,41 +1,41 @@
 import express from 'express';
 const app = express();
 
-app.get('', async (req, res) => {
+app.get('', async (req: express.Request, res: express.Response) => {
```
Comment (florian-richter) [inline samples/http-client-examples/server.ts:4]:
```suggestion
const { Request, Response } = express;

app.get('', async (req: Request, res: Response) => {
```
[pp]

**Review by florian-richter — APPROVED** [review]
I prefer to extract the types so it's a bit shorter. But either way is fine.

File: samples/http-client-examples/server.ts (hunk 1 above)
Comment (fwilhe) [reply samples/http-client-examples/server.ts:4]: I've configured auto-merge so I could not address your comment, I don't have a strong opinion on that. Honestly I was not aware of this syntax.

Comment (marikaner) [reply samples/http-client-examples/server.ts:4]: Wanted to say +1 on florians comment, but I would even prefer this:
`import { Request, Response } from "express";`

**Pull request merged**